code/modules/surgery/surgery_initiator.dm

```diff
@@ -9,6 +9,10 @@ proc/initiate_surgery_moment(obj/item/tool, mob/living/carbon/target, obj/limb/a
 	var/list/available_surgeries = list()
 	var/list/valid_steps = list() //Steps that could be performed, if we had the right tool.
 
+	if(user.z == GLOB.interior_manager.interior_z && !istype(target.loc, /turf/open/shuttle/vehicle/med))
```

Comment:
a better solution might be a var on turfs that goes like "supports_surgery" true or false. check if the loc is a turf, then check if it supports surgery